content/user/ui/report/Vulnerability-trends.md

```diff
@@ -0,0 +1,71 @@
+<!--
+title: "Vulnerability Trends"
+description: "Overview of vulnerability trend reports"
+tags: "TeamServer application vulnerability PDF report"
+-->
+ 
+Vulnerability management is a vital responsibility of any security team. Recognizing the vulnerabilities your applications face and how well they're being managed can help you understand your security posture.
+
+The report presents data about newly discovered vulnerabilities as well as accumulated total vulnerabilities in comparison to remediations. Focus on the details that you care about by using robust filtering options to look at data over any applications, servers, Assess rules or by vulnerability severity. Contrast can also present data in various timeframes, ranging from the last week to the last year. The customized results allow you to prioritize efforts toward vulnerability remediation.
+
+## Access Data
+ 
+Select **Reports** in the User menu to go the Vulnerability Trend dashboard. Click the **View** link to see the graphs in more detail. 
+
+Select **New** to see a graph of new vulnerabilities. Select **Total** to see a graph of all reported vulnerabilities compared to all remediated vulnerabilities. Each point in the graph generates a tooltip with the number of suspicious, confirmed and reported vulnerabilities on that date when you hover over it. 
+
+<a href="assets/images/Vulnerability-trend-new.png" rel="lightbox" title="Graph of new vulnerabilities"><img class="thumbnail" src="assets/images/Vulnerability-trend-new.png"/></a>
+
+<a href="assets/images/Vulnerability-trend-total.png" rel="lightbox" title="Graph of total vulnerabilities"><img class="thumbnail" src="assets/images/Vulnerability-trend-total.png"/></a>
+
+###Filter vulnerabilities
+
+Each report defaults to all applications, servers and rules, but you can filter vulnerabilities by clicking in the fields above the graph. The following table outlines the categories that you can use to create a custom report. 
+
+
+| Field        | Default      | Filter Options                                                                                              |
+|--------------|--------------|-------------------------------------------------------------------------------------------------------------|
+| Date         | Last 7 Days  | Last 30 days <br> Last 12 weeks <br> Last 12 months                                                                   |
+| Applications | All          | Importance (Critical, High, Medium, Low, Unimportant) <br> Application Tags <br> Licensed (List of all applications) |
+| Servers      | All          | Environment (Development, QA, Production) <br> Server Tags <br> Servers (List of all servers)                         |
+| Rules        | All          | Severity (Critical, High, Medium, Low, Note) <br> Vulnerability Tags <br> Vulnerability Rules (List of all rules)                         |
+
+
+<!-- Add info re: build number option - App > Vuln trend -->
```

Comment:
@terra4383 It looks like the option to go to the full Vulnerability Trend from the application is either not working or not finished, so will wait...unless you think that I should just reference the build number option in the meantime. Thoughts?